Makefile

```diff
@@ -150,7 +169,8 @@ clean:
 	# ensure docker is running and we can talk to it, abort if not:
 	docker ps > /dev/null
 	rm -f misc/certs/ca-certificates.crt &> /dev/null
-	rm -f out/amazon-ecs-agent out/amazon-ecs-agent.exe &> /dev/null
+	rm -rf out/*
+	rm -rf $(shell pwd)/amazon-ecs-cni-plugins
```

Comment:
Just choosing to use the absolute path to remove the sources dir here. 
We could also use relative paths but I prefer the current approach.